backend/src/main/scala/cromwell/backend/standard/callcaching/RootWorkflowFileHashCacheActor.scala

```diff
@@ -80,8 +80,6 @@ class RootWorkflowFileHashCacheActor private[callcaching](override val ioActor:
       case FileHashValueNotRequested =>
         log.info(msgIoAckWithNoRequesters.format(fileHashContext.file))
         notifyRequestersAndCacheValue(List.empty[FileHashRequester])
-      case _ =>
```

Comment:
I think a catch-all "we didn't expect that message" is still useful. If there are message types we now know that we could expect to see but which require no action, I think we should add `case Foo => # no-op` above this instead of deleting it.